frontend/components/text-classifier/results/ClassifierAnnotationArea.vue

```diff
@@ -181,103 +161,59 @@ export default {
     annotate() {
       this.$emit("validate", { labels: this.selectedLabels });
     },
-    onVisibility(visible) {
-      this.visible = visible;
+    showHiddenLabels() {
+      this.maxLabels = this.filteredLabels.length;
+    },
+    hideHiddenLabels() {
+      this.maxLabels = 7;
     },
-    decorateScore(score) {
-      return score * 100;
+    onSearchLabel(event) {
+      this.searchText = event;
     },
   },
 };
 </script>
 <style lang="scss" scoped>
 %item {
-  width: 30%;
-  min-width: 225px;
-  flex-grow: 0;
-  flex-shrink: 0;
-  margin-left: 1% !important;
-  margin-right: 1% !important;
+  // width: calc(25% - 5px);
+  min-width: 80px;
   max-width: 238px;
 }
 .feedback-interactions {
   margin: 1.5em auto 0 auto;
   padding-right: 0;
-  & > div {
-    width: 100%;
-  }
+  // & > div {
+  //   width: 100%;
+  // }
```

Comment:
```suggestion
```

Reply:
@frascuchon waiting to choose a style for the labes (fix width or natural width) to eliminate it